includes/functions.php

```diff
@@ -238,8 +238,11 @@ function is_tombstone( $wp_error ) {
 function get_rest_url_by_path( $path = '' ) {
 	// we'll handle the leading slash.
 	$path = ltrim( $path, '/' );
-	$url = sprintf( '/%s/%s', ACTIVITYPUB_REST_NAMESPACE, $path );
-	return \get_rest_url( null, $url );
+	$namespaced_path = sprintf( '/%s/%s', ACTIVITYPUB_REST_NAMESPACE, $path );
+	$rest_url = \get_rest_url( null, $namespaced_path );
+	// Just in case there are non-default ways of handling REST URLs.
+	$rest_url = \apply_filters( 'activitypub_rest_url', $rest_url, $path, ACTIVITYPUB_REST_NAMESPACE );
```

Comment:
do we need this filter? `get_rest_url` already has a filter that might work already: https://developer.wordpress.org/reference/hooks/rest_url/

Reply:
We may not. I was mostly expecting it to already be transformed and it wasn't so I reached for this first. Further investigation needed